Optik/Classes/ImageViewController.swift

```diff
@@ -30,7 +30,8 @@ internal final class ImageViewController: UIViewController {
         }
     }
     private(set) var imageView: UIImageView?
-
+    var likeButton: UIButton?
```

Comment:
overall the like button should work like the dismiss button, the code should be very similar in every way. Doesn't look like it's the case.